code/controllers/subsystem/persistence.dm

```diff
@@ -0,0 +1,105 @@
+//Season defines, universal for all seasons
+#define LAST_UPDATE "last_update" //last time the season changed
+#define CURRENT_SEASON "current_season" //current season of the section
```

Comment:
this way of doing is weird, but i guess it's from OG persistence SS

Reply:
discussed on discord that this is ok for now